README.md

```diff
@@ -77,21 +77,21 @@ try {
 }
 ```
 
-**get all entries**
+**get all instances**
 
 ```js
-const found = await client.modelFind(this.finderID, 'Folder2P', {})
-console.debug('found:', found.entitiesList.length)
-this.folders = found.entitiesList.map((entity) => entity).map((obj) => {
+const found = await client.instanceFind(this.finderID, 'Folder2P', {})
+console.debug('found:', found.instancesList.length)
+this.folders = found.instancesList.map((instance) => instance).map((obj) => {
```

Comment:
All these crud methods should have no prefix at all now. Just `find`, `list`, etc